pkg/destroy/bootstrap/bootstrap.go

```diff
@@ -41,7 +41,7 @@ func Destroy(dir string) (err error) {
 	// Set platform to azurestack after setting tfPlatformVarsFileName, because the
 	// tfvars file is still terraform.azure.auto.tfvars.json in the case of Azure Stack.
```

Comment:
Not for this PR, but we should consider whether there is any benefit in using a different name for the tfvars file for each platform. We never have tfvars for multiple platforms simultaneously, so we could just always use the name `terraform.platform.auto.tfvars.json` for the file.

Reply:
https://github.com/openshift/installer/pull/5477